notebooks/13_Theta_phase_and_power.ipynb

```diff
@@ -0,0 +1,1123 @@
+{
```

Comment:
<p>It may be worth showing them how to check if the LFPBand data exist for the electrodes they're using. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I think this is a quite late tutorial so people should know how to do that already? Just trying to keep the tutorial not too complicated but can add it if definitely necessary.